Templates/Common-Backend-Scripts/zBuilder.sh

```diff
@@ -0,0 +1,542 @@
+    echo "                             perform build with options for   "
+    echo "                             performance optimized            "
+    echo "                             executables                      "
+    echo "                            *preview* -                       "
+    echo "                             perform build build in           "
+    echo "                             preview mode                     "
+    echo "                                                              "
+    echo "                 Ex: build                                    "
+    echo "                                                              "
+    echo "       -q                 - (Optional) dataset qualifier      "
+    echo "                             for build datasets               "
+    echo "                             to override the configured       "
+    echo "                             dataset qualifer in              "
+    echo "                             pipelineBackend.config           "
+    echo "                 Ex: USER.APP                                 "
+    echo "                                                              "
+    echo "       -t                 - (Optional) zBuilder build         "
+    echo "                             lifecycle  override              "
+    echo "                                                              "
+    echo "                 Ex: -t 'full'                                "
+    echo "                                                              "
+    echo "       -v                 - (Optional) Verbose tracing        "
+    echo "                                                              "
+    echo "                                                              "
+    exit 0
+
+}
+
+#
+# Configuration file leveraged by the backend scripts
+# Either an absolute path or a relative path to the current working directory
+SCRIPT_HOME="$(dirname "$0")"
+pipelineConfiguration="${SCRIPT_HOME}/pipelineBackend.config"
+buildUtilities="${SCRIPT_HOME}/utilities/dbbzBuilderUtils.sh"
+
+
+#
+# Internal Variables
+#set -x                  # Uncomment to enable shell script debug
+#export BASH_XTRACEFD=1  # Write set -x trace to file descriptor
+
+PGM=$(basename "$0")
+PGMVERS="1.00"
+USER=$USER
+SYS=$(uname -Ia)
+
+rc=0
+ERRMSG=""
+
+# Initialized option variables passed to this script
+Workspace=""
+App=""
+Branch=""
+PipelineType=""
+HELP=$1
+
+# Local Variables
+AppDir=""                  # Derived Application Directory
+HLQ=""                     # Derived High Level Qualifier
+HLQPrefix=""               # Prefix of HLQ, either specified via the cli option -q or via configuration file
+outDir=""                  # Computed output directory to store build protocols
+nestedApplicationFolder="" # Flag to understand a nested repository
+Lifecycle=""               # Derived zBuilder lifecycle type
```

Comment:
There is a mixed case in the variable names, starting with uppercase and lowercase. Can we have a uniform approach? I know it will require some work.